### PR TITLE
add storage methods to java pec

### PR DESCRIPTION
**Pull request by mmandlis**

### CODE REVIEW

File: src/javaharness/java/arcs/api/PECInnerPortImpl.java [hunk 1]
```diff
@@ -155,6 +160,60 @@ public void SynchronizeProxy(StorageProxy storageProxy, Consumer<PortableJson> c
         postMessage(message);
     }
 
+    @Override
+    public void HandleStore(StorageProxy storageProxy, Consumer<PortableJson> callback, PortableJson data, String particleId) {
+        PortableJson message = jsonParser.parse("{}");
```
Comment (shaper) [inline src/javaharness/java/arcs/api/PECInnerPortImpl.java]: Maybe create a helper for this and use it everywhere? Or, add a "createEmptyMessage" method on jsonParser itself?

File: src/javaharness/java/arcs/api/PECInnerPortImpl.java [hunk 2]
```diff
@@ -155,6 +160,60 @@ public void SynchronizeProxy(StorageProxy storageProxy, Consumer<PortableJson> c
...
+        body.put(PROXY_HANDLE_ID_FIELD, mapper.identifierForThing(new Thing<StorageProxy>(storageProxy)));
+        body.put(PROXY_CALLBACK_FIELD, mapper.createMappingForThing(
+            new Thing<Consumer<PortableJson>>(callback), /* requestedId= */ null));
+        message.put(MESSAGE_BODY_FIELD, body);
+        postMessage(message);
+    }
+
+    @Override
+    public void HandleRemove(StorageProxy storageProxy, Consumer<PortableJson> callback, PortableJson data, String particleId) {
+        PortableJson message = jsonParser.parse("{}");
```
Comment (shaper) [inline src/javaharness/java/arcs/api/PECInnerPortImpl.java]: This method impl is identical to RemoveMultiple except for the message type, suggest adding helper they both use just to make the code duplication more apparent.

File: src/javaharness/java/arcs/api/PECInnerPortImpl.java (hunk 1 above)
Comment (mmandlis) [reply src/javaharness/java/arcs/api/PECInnerPortImpl.java]: done (added emptyJson to parser)

File: src/javaharness/java/arcs/api/PECInnerPortImpl.java (hunk 2 above)
Comment (mmandlis) [reply src/javaharness/java/arcs/api/PECInnerPortImpl.java]: thanks, done (plus some more refactoring)

**Comment by mmandlis** [comment]
addressed comments, PTAL

**Pull request merged**